components/brave_wallet_ui/page/screens/send/send/send.tsx

```diff
@@ -364,16 +387,24 @@ export const Send = (props: Props) => {
       showEnsOffchainWarning
   }, [toAddressOrUrl, searchingForDomain, showEnsOffchainWarning])
 
+  const { userVisibleTokensInfo } = useGetUserTokensRegistryQuery(undefined, {
+    selectFromResult: result => ({
+      userVisibleTokensInfo: selectAllUserAssetsFromQueryResult(result)
```

Comment:
I beleive this is all tokens, not just visible tokens. You probably need to define a new selector for visible assets filtering.

Reply:
Can you help me with some example code for the above two remarks? Thanks in advance!

Reply:
Fixed here: https://github.com/brave/brave-core/pull/19337/commits/4834bb770ac63b25a866ec2413fc7f3b6834d084